js/packages/components/dropdowns/members/MembersDropdown.tsx

```diff
@@ -18,9 +22,59 @@ interface MembersDropdownProps {
 	accessibilityLabel?: string
 }
 
+interface MemberItemProps {
+	onPress: () => void
+	item: beapi.messenger.IMember
+	convPK: string
+}
+
+const MemberItem: React.FC<MemberItemProps> = ({ onPress, convPK, item }) => {
+	const { padding } = useStyles()
+	const dispatch = useAppDispatch()
+	const [peer, setPeer] = React.useState<PeerNetworkStatus | null>(null)
+
+	React.useEffect(() => {
+		const f = async () => {
+			const _peer = await dispatch(
+				getPeerFromMemberPK({ memberPK: item.publicKey, convPK: item.conversationPublicKey }),
+			)
+			setPeer(_peer.payload as PeerNetworkStatus)
+		}
+
+		f()
+	}, [convPK, dispatch, item.publicKey, item.conversationPublicKey])
+
+	return (
+		peer && (
+			<TouchableOpacity onPress={onPress} style={[styles.item, padding.horizontal.medium]}>
+				<View style={[styles.content]}>
+					<MemberAvatarWithStatus
+						publicKey={item.publicKey}
+						convPK={convPK}
+						memberStatus={peer.connectionStatus}
+					/>
+					<MemberName displayName={item.displayName} />
+				</View>
+				<MemberTransport memberStatus={peer.connectionStatus} memberTransport={peer.transport} />
+			</TouchableOpacity>
+		)
+	)
+}
+
 export const MembersDropdown: React.FC<MembersDropdownProps> = props => {
-	const { padding, margin, border } = useStyles()
+	const { border } = useStyles()
 	const colors = useThemeColor()
+	const [value, setValue] = React.useState<string>('')
+
+	const lowSearchValue = value.toLowerCase()
+	const searchCheck = React.useCallback(
+		searchIn => searchIn.toLowerCase().includes(lowSearchValue),
+		[lowSearchValue],
+	)
+	const searchMembers = React.useMemo(
+		() => (value.length ? pickBy(props.items, val => searchCheck(val.displayName)) : props.items),
+		[props.items, searchCheck, value],
+	)
```

Comment:
inlining searchCheck instead of using an useCallback will be more performant I believe